### ISSUE TITLE
iOS 13 Accessibility: Address bar difficult to access with voice control

### ISSUE THREAD

**Issue by MarcoZehe** [mozilla-mobile/firefox-ios#5495]
The UIAccessibilityLabel for the address bar is currently dynamically set to the address that's loaded in a tab. However, since the new voice control feature in iOS 13 uses the AccessibilityLabel for tapping on things, like "Tap Address Bar", this should become a permanent (localized) name. The URL of the currently loaded page should go in the UIAccessibilityValue property instead. This is standard for text fields anyway.

This will also make VoiceOver speech more friendly. These technologies all use the same APIs.

CC @jcsteh

**Comment by MarcoZehe** [comment]
And [here is a blog post](https://kristina.io/adopting-voice-control/) explaining how voice control labels are being applied to controls if the AccessibilityLabel for VoiceOver does somehow not fit. But again, there are actually two fixes needed here: One is an accessibilityLabel for the address bar in the first place, and second to check whether that also makes sense for Voice Control.
